packages/back-end/src/api/sdk-connections/postSdkConnection.ts

```diff
@@ -0,0 +1,80 @@
+import { PostSdkConnectionResponse } from "../../../types/openapi";
+import {
+  toApiSDKConnectionInterface,
+  createSDKConnection,
+} from "../../models/SdkConnectionModel";
+import { createApiRequestHandler } from "../../util/handler";
+import { postSdkConnectionValidator } from "../../validators/openapi";
+import { findAllProjectsByOrganization } from "../../models/ProjectModel";
+import { sdkLanguages } from "../../util/constants";
+import { SDKLanguage } from "../../../types/sdk-connection";
+
+export const postSdkConnection = createApiRequestHandler(
+  postSdkConnectionValidator
+)(
+  async (req): Promise<PostSdkConnectionResponse> => {
+    const {
+      name,
+      languages,
+      sdkVersion,
+      environment,
+      projects = [],
+      encryptPayload = false,
+      includeVisualExperiments = false,
+      includeDraftExperiments = false,
+      includeExperimentNames = false,
+      includeRedirectExperiments = false,
+      proxyEnabled,
+      proxyHost,
+      hashSecureAttributes = false,
+      remoteEvalEnabled,
+    } = req.body;
+
+    if (name.length < 3) {
+      throw Error("Name length must be at least 3 characters");
+    }
+
+    if (projects) {
+      const allProjects = await findAllProjectsByOrganization(req.context);
+      const nonexistentProjects = projects.filter(
+        (p) => !allProjects.some(({ id }) => p === id)
+      );
+      if (nonexistentProjects.length)
+        throw new Error(
+          `The following projects do not exist: ${nonexistentProjects.join(
+            ", "
+          )}`
+        );
+    }
+
+    if (!languages.length)
+      throw new Error("You need to specify some lanuages!");
+
+    languages.forEach((lang) => {
+      if (!(sdkLanguages as readonly string[]).includes(lang))
+        throw new Error(`Language ${lang} is not supported!`);
+    });
```

Comment:
We should default the `sdkVersion` to the latest version using `getLatestSDKVersion(language)`.

We also might want to check version compatibility here.  Like only some languages/versions support remoteEval.

```ts
const capabilities = getSDKCapabilities(language, sdkVersion);

if (remoteEvalEnabled && !capabilities.includes("remoteEval")) {
  // Throw error
}
```

This capability checking also applies to the PUT endpoint